src/Core/src/Platform/iOS/ModalWrapper.cs

```diff
@@ -0,0 +1,8 @@
+﻿using UIKit;
+
+namespace Microsoft.Maui.Platform
+{
+	internal class ModalWrapper : UIViewController
```

Comment:
Should this rather be an interface if it does not do anything specific?

Reply:
Perhaps eventually? 

At some point we'll move modals into core and those APIs will all get designed.  There's not really a larger use case here yet for an interface so I don't think we need to make that leap yet.